.ci/job_matrix.yaml

```diff
@@ -55,8 +58,20 @@ steps:
       docker pull ${DOCKER_IMAGE_NAME}
       docker create -ti --rm $DOCKER_OPT ${DOCKER_IMAGE_NAME} /bin/bash > ${WORKSPACE}/ucc_docker.id
       docker start $(cat ${WORKSPACE}/ucc_docker.id)
-
+      
+  #============================================================================
+  - name: Run Coverity
+    credentialsId: 'bc9a18d3-1153-449c-b924-7fc9249c9cc0'
```

Comment:
Duplicate?

Reply:
no, need to add what steps use this credential ID (according to ci-demo )